Security/ipsAnywhere.md

```diff
@@ -0,0 +1,70 @@
+{{{ "title": "IPS - CLI Install & Uninstall",
+        "date": "1-11-2016",
+        "author": "Client-Security",
+        "attachments": [],
+        "contentIsHTML": false,
+        "sticky": false }}}
+
+# IPS-API
```

Comment:
Remove this.  The title will be used as the top header automatically.  Also, `###` is the smallest header number used in kb articles.